### PR TITLE
update js snippet to sv5

### PR DESCRIPTION
**Pull request by TimothyMothra**
Supersedes #2507 

## Changes
- Update JS Snippet to sv5
- Added JavaScriptSnippet to example project for ease of testing.
  I personally verified that the snippet still works after this update.

@MSNev please review

### Checklist
- [ ] I ran Unit Tests locally.
- [ ] CHANGELOG.md updated with one line description of the fix, and a link to the original issue if available.

For significant contributions please make sure you have completed the following items:

- [ ] Design discussion issue #
- [ ] Changes in public surface reviewed

The PR will trigger build, unit tests, and functional tests automatically. Please follow [these](https://github.com/Microsoft/ApplicationInsights-dotnet/blob/develop/.github/CONTRIBUTING.md) instructions to build and test locally.

### Notes for authors:
- FxCop and other analyzers will fail the build. To see these errors yourself, compile localy using the Release configuration.

### Notes for reviewers:

- We support [comment build triggers](https://docs.microsoft.com/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#comment-triggers)
  - `/AzurePipelines run` will queue all builds
  - `/AzurePipelines run <pipeline-name>` will queue a specific build

### CODE REVIEW

File: NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/Resources.resx [hunk 1]
```diff
@@ -121,11 +121,15 @@
...
+src: "https://js.monitor.azure.com/scripts/b/ai.2.min.js", // The SDK URL Source
+// name: "appInsights", // Global SDK Instance name defaults to "appInsights" when not supplied
+// ld: 0, // Defines the load delay (in ms) before attempting to load the sdk. -1 = block page load and add to head. (default) = 0ms load after timeout,
```
Comment (MSNev) [inline NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/Resources.resx:127]: A couple of minor nit's
Lets remove the commented out lines

Suggestion: Uncomment and keep the crossOrigin one.

**Review by MSNev — APPROVED** [review]
Approved with some minor suggestions

File: NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/Resources.resx (hunk 1 above)
Comment (TimothyMothra) [reply NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/Resources.resx:127]: Just to verify;

- Lines 126, 127, 128 - delete
- Lines 129 - uncomment
- Lines 130 - delete

is this correct?

Comment (MSNev) [reply NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/Resources.resx:127]: Correct 😄

**Review by MSNev — APPROVED** [review]

**Review by MSNev — APPROVED** [review]

**Review by cijothomas — APPROVED** [review]

**Pull request merged**